helm/prometheus-rules/templates/platform/atlas/recording-rules/signals.observability-platform.rules.yml

```diff
@@ -0,0 +1,119 @@
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  labels:
+    {{- include "labels.common" . | nindent 4}}
+  name: signals.observability-platform.rules
+  namespace: {{ .Values.namespace }}
+spec:
+  groups:
+    - name: signals.observability-platform
+      rules:
+        # Mimir rules with extracted cluster_id and enriched labels
+        - record: giantswarm:observability:signals:mimir_rules
+          expr: |
+            sum by (cluster_id, cluster_type, customer, installation, pipeline, provider, region, source_system, creation_mode, tenant) (
+              label_replace(
+                label_replace(
+                  label_replace(
+                    label_replace(
+                      label_replace(
+                        sum by (cluster_id, cluster_type, customer, installation, pipeline, provider, region, user, rule_group) (
+                          cortex_prometheus_rule_group_rules{cluster_type="management_cluster", rule_group=~".+"}
+                        ),
+                        "cluster_id", "$1", "rule_group", "/data/[^/]+/([^%/]+)%2F[^%/]+%2F[^%/]+%2F[^;]+;.*"
+                      ),
+                      "source_system", "mimir", "", ""
```

Comment:
Why adding `source_system`, it's already in the recorded metrics name: `mimir_rules` right?

Maybe they could all go in a generic metrics named `giantswarm:observability:signals:rules` and be distinct from each other via the `source_system` label?